_overviews/scala3-book/taste-functions.md

```diff
@@ -22,33 +21,44 @@ The `map` method of the `List` class is a typical example of a higher-order func
 
 These two examples are equivalent, and show how to multiply each number in a list by `2` by passing a lambda into the `map` method:
 
+
+{% tabs function_1 class=tabs-scala-version %}
```

Comment:
`class=tabs-scala-version` should not be used for "Scala 2 and 3" tabs. Soon I am adding assertions to make CI fail for this
```suggestion
{% tabs function_1 %}
```